Objects/longobject.c

```diff
@@ -4498,28 +4488,49 @@ long_rshift(PyLongObject *a, PyLongObject *b)
 }
 
 static PyObject *
-long_lshift(PyObject *v, PyObject *w)
+long_rshift(PyObject *a, PyObject *b)
 {
-    /* This version due to Tim Peters */
-    PyLongObject *a = (PyLongObject*)v;
-    PyLongObject *b = (PyLongObject*)w;
-    PyLongObject *z = NULL;
-    Py_ssize_t oldsize, newsize, wordshift, i, j;
+    Py_ssize_t wordshift;
     digit remshift;
-    twodigits accum;
 
     CHECK_BINOP(a, b);
 
     if (Py_SIZE(b) < 0) {
-        PyErr_SetString(PyExc_ValueError, "negative shift count");
+        PyErr_SetString(PyExc_ValueError,
```

Comment:
Nitpick: why the reformatting here, but not in the same line in `long_lshift`?

Reply:
It was not reformatted. The original code in `long_rshift()` was split on two lines, while in `long_lshift()` it was one line. It is GidHub that shows the diff with wrong functions because some code was moved.

Reply:
Makes sense. Thanks.